utils/display/flink_jobs_test.go

```diff
@@ -85,8 +85,8 @@ func verifyPrintFlinkJobsOutput(t *testing.T, out string, flinkJob *meroxa.Flink
 	if !strings.Contains(out, flinkJob.UUID) {
 		t.Errorf("%s, not found", flinkJob.UUID)
 	}
-	if !strings.Contains(out, string(flinkJob.Status.State)) {
-		t.Errorf("state %s, not found", flinkJob.Status.State)
+	if !strings.Contains(out, string(flinkJob.Status.LifecycleState)) {
```

Comment:
Do I understand it correctly, that for this phase of the work `LifecycleState` is the best representation of a flink job's state? Is it, because `State` will not be populated for now since the state computation will be implemented in a later stage of the project?

Reply:
That's our best guess for the time being, and it may change soon.  I am looking forward to having one calculated state value, but we just don't know enough to do that yet.